exercises/bob/bob_test.rb

```diff
@@ -2,133 +2,184 @@
   end
 
   def test_shouting_numbers
     skip
-    remark = '1, 2, 3 GO!'
-    assert_equal 'Whoa, chill out!', bob.hey(remark), feedback(remark)
+    remark = "1, 2, 3 GO!"
+    assert_equal 'Whoa, chill out!', Bob.hey(remark), "Bob hears 1, 2, 3 GO!, and.."
   end
 
   def test_only_numbers
     skip
-    remark = '1, 2, 3'
-    assert_equal 'Whatever.', bob.hey(remark), feedback(remark)
+    remark = "1, 2, 3"
+    assert_equal 'Whatever.', Bob.hey(remark), "Bob hears 1, 2, 3, and.."
   end
 
   def test_question_with_only_numbers
     skip
-    remark = '4?'
-    assert_equal 'Sure.', bob.hey(remark), feedback(remark)
+    remark = "4?"
+    assert_equal 'Sure.', Bob.hey(remark), "Bob hears 4?, and.."
   end
 
   def test_shouting_with_special_characters
     skip
-    remark = 'ZOMG THE %^*@#$(*^ ZOMBIES ARE COMING!!11!!1!'
-    assert_equal 'Whoa, chill out!', bob.hey(remark), feedback(remark)
+    remark = "ZOMG THE %^*@#$(*^ ZOMBIES ARE COMING!!11!!1!"
+    assert_equal 'Whoa, chill out!', Bob.hey(remark), "Bob hears ZOMG THE %^*@#$(*^ ZOMBIES ARE COMING!!11!!1!, and.."
   end
 
   def test_shouting_with_no_exclamation_mark
     skip
-    remark = 'I HATE YOU'
-    assert_equal 'Whoa, chill out!', bob.hey(remark), feedback(remark)
+    remark = "I HATE YOU"
+    assert_equal 'Whoa, chill out!', Bob.hey(remark), "Bob hears I HATE YOU, and.."
   end
 
   def test_statement_containing_question_mark
     skip
-    remark = 'Ending with ? means a question.'
-    assert_equal 'Whatever.', bob.hey(remark), feedback(remark)
+    remark = "Ending with ? means a question."
+    assert_equal 'Whatever.', Bob.hey(remark), "Bob hears Ending with ? means a question., and.."
+  end
+
+  def test_non_letters_with_question
+    skip
+    remark = ":) ?"
+    assert_equal 'Sure.', Bob.hey(remark), "Bob hears :) ?, and.."
   end
 
   def test_prattling_on
     skip
-    remark = 'Wait! Hang on. Are you going to be OK?'
-    assert_equal 'Sure.', bob.hey(remark), feedback(remark)
+    remark = "Wait! Hang on. Are you going to be OK?"
+    assert_equal 'Sure.', Bob.hey(remark), "Bob hears Wait! Hang on. Are you going to be OK?, and.."
   end
 
   def test_silence
     skip
-    remark = ''
-    assert_equal 'Fine. Be that way!', bob.hey(remark), feedback(remark)
+    remark = ""
+    assert_equal 'Fine. Be that way!', Bob.hey(remark), "Bob hears , and.."
   end
 
   def test_prolonged_silence
     skip
-    remark = ' ' * rand(1..10)
-    assert_equal 'Fine. Be that way!', bob.hey(remark), feedback(remark)
+    remark = "          "
+    assert_equal 'Fine. Be that way!', Bob.hey(remark), "Bob hears           , and.."
   end
 
-  def test_alternate_silences
+  def test_alternate_silence
     skip
-    remark = "\t" * rand(1..10)
-    assert_equal 'Fine. Be that way!', bob.hey(remark), feedback(remark)
+    remark = "										"
+    assert_equal 'Fine. Be that way!', Bob.hey(remark), "Bob hears 										, and.."
```

Comment:
The things that Bob hears should be quoted.
`%Q` can be good for this to help avoid needing to quote quotes.